lib/parse.js

```diff
@@ -68,7 +68,7 @@ module.exports = (input) => {
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/g.test(line);
+        return /^(\s\s[-\s]\s)?[a-zA-Z]+:/g.test(line);
```

Comment:
before parsing the output with yaml, this filter is applied, so it removes the line I added earlier.
"    volumeName: "